engine/crates/engine/src/registry/mod.rs

```diff
@@ -1421,6 +1421,8 @@ pub struct Registry {
     pub federation_entities: BTreeMap<String, FederationEntity>,
     #[serde(default)]
     pub enable_ai: bool,
+    #[serde(default)]
+    pub grafbase_cli_version: Option<String>,
```

Comment:
Alternatively, the version could be outside the registry, the CLI would write an object like `type RegistryWithVersion = { version: string, registry: Registry }`. Maybe that would be cleaner, do we need the version in the registry in other cases? What do you think?

Reply:
I'm going to make that change, I don't like how intrusive this PR is with the registry.

Reply:
Done.